common/app/common/configuration.scala

```diff
@@ -21,11 +24,42 @@ class GuardianConfiguration extends Logging {
   case class OAuthCredentials(oauthClientId: String, oauthSecret: String, oauthCallback: String)
   case class OAuthCredentialsWithMultipleCallbacks(oauthClientId: String, oauthSecret: String, authorizedOauthCallbacks: List[String])
 
-  protected val configuration = ConfigurationFactory.getNonLoggingConfiguration("frontend", "env")
+  import com.gu.cm.{Configuration => CM}
+  import com.typesafe.config.Config
+  val configuration = {
 
-  private implicit class OptionalString2MandatoryString(conf: com.gu.conf.Configuration) {
-    def getMandatoryStringProperty(property: String) = configuration.getStringProperty(property)
+    val cmLogger = new Logger {
```

Comment:
There is a class that provides that implementation in the play module: `PlayDefaultLogger`. It's named play-24 but don't be afraid to use it as it works with play 2.5